src/layers/transform/concatenate.cpp

```diff
@@ -326,5 +326,6 @@ void bp_compute_impl(
 
 #define LBANN_INSTANTIATE_CPU_HALF
 #include "lbann/macros/instantiate.hpp"
+#undef PROTO
```

Comment:
I thought that we didn't need to do this in cpp files.

Reply:
We don't. Will remove. (Artifact of some temporary changes)